web_sdk/BUILD.gn

```diff
@@ -144,6 +145,13 @@ sdk_rewriter("web_unicode_library") {
   output_dir = "$root_out_dir/flutter_web_sdk/lib/_web_unicode/"
 }
 
+sdk_rewriter("web_test_fonts_library") {
+  library_name = "web_test_fonts"
+  api_file = "//flutter/third_party/web_test_fonts/lib/web_test_fonts.dart"
+  input_dir = "//flutter/third_party/web_test_fonts/lib/web_test_fonts/"
```

Comment:
This package needs to exist. For example, look at:

- https://github.com/flutter/engine/tree/main/third_party/web_unicode
- https://github.com/flutter/engine/tree/main/third_party/web_locale_keymap

Perhaps you forgot to include them in the commit?

Reply:
Sorry forgot to track those files. The tests are passing now.